src/components/pagination/pagination.scss

```diff
@@ -167,12 +172,21 @@ $indicator-width-current: $indicator-height-current;
 }
 
 .c-pagination__action.is-next::after {
-  background-image: svg-load(
-    'icons/arrow-right.svg',
-    stroke=colors.$primary-brand
-  );
   right: 0;
 
+  /**
+   * Inline SVGs aren't aware of custom properties, so we have to do a bit
+   * of theming trickery here
+   */
+
+  &,
+  .t-light & {
+    background-image: svg-load(
+      'icons/arrow-right.svg',
+      stroke=colors.$primary-brand
+    );
+  }
+
```

Comment:
Nit: this pattern has been used three times and will likely be used more. I wonder if it would be worth wrapping in a mixin for easier use? (Otherwise I'll be searching these out and copy/pasting them whenever I need to do something similar)

Reply:
Hmm, I'm torn on this.

On one hand, I see what you mean. We're using this pattern a few different times.

On the other hand, I sort of _want_ to discourage this method of theming over the user of custom properties. And the only part we'd really be keeping DRY is `&, .t-light &`, which feels leaner than any mixin name might be.

Reply:
That's fair. I don't feel strongly about this.